pkg/api/create_person_handler.go

```diff
@@ -0,0 +1,49 @@
+package api
+
+import (
+	"encoding/json"
+
+	"github.com/darchlabs/api-example/pkg/person"
+	"github.com/go-playground/validator/v10"
+)
+
+type CreatePersonHandlerInputs struct {
```

Comment:
I think that you do not use this type of implementation on anywhere of PR

Reply:
Got it, already changed